endpoints/cookie_sync.go

```diff
@@ -125,6 +132,21 @@ func (c *cookieSyncEndpoint) parseRequest(r *http.Request) (usersync.Request, pr
 		}
 	}
 
+	if request.Account != "" {
+		accountInfo, errs := accountService.GetAccount(context.Background(), c.config, c.accountsFetcher, request.Account)
+		if len(errs) > 0 {
+			return usersync.Request{}, privacy.Policies{}, errs[0]
+		}
+		if request.Limit == 0 {
+			request.Limit = accountInfo.Defaults.DefaultLimit
+		} else if request.Limit > accountInfo.Defaults.MaxLimit {
+			request.Limit = accountInfo.Defaults.MaxLimit
+		}
+		if request.CooperativeSync == nil {
+			request.CooperativeSync = &accountInfo.Defaults.DefaultCoopSync
+		}
+	}
```

Comment:
I like the tests you have in `TestNewCookieSyncEndpoint` for end to end functionality. Please also add tests to `TestCookieSyncParseRequest` which can include additional permutations, such as handling a potential error from the account fetcher on line 136.